doc/usage.rst

```diff
@@ -98,6 +98,12 @@ few command line options:
    to parse ``setup.py``. This flag can be provided multiple times to
    pass multiple packages for pre-install.
 
+.. cmdoption:: --pip-tool <exename>
+
+   Executable that will be used to install requirements after the
+   preinstall stage.  Usually you'll install this program by using the
+   ``--preinstall`` argument. By default vanilla pip is used.
```

Comment:
"Vanilla pip" is bit vague, maybe this could be reworded that by default, the pip supplied with the virtualenv is used?